reference/orders.v3.yml

```diff
@@ -2628,7 +2630,7 @@ components:
         amount:
           type: number
           format: float
-          description: 'A negative 2 decimal place rounded value to deduct from the amount refunded.'
+          description: 'A negative 2 decimal place rounded value to deduct from the amount refunded. When creating refunds and refund quotes, this field becomes irrelevant when you select PRODUCT for `item_type`.'
```

Comment:
I don't think this change is correct because this is "amount" of `RefundItemAdjustment` and it should be relevant for "PRODUCT" item_type